### DISCUSSION TITLE
Welcome to vuepress-theme-reco Discussions!

### DISCUSSION THREAD

**Discussion by recoluan** [vuepress-reco/vuepress-theme-reco-1.x#280] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LoEnzo** [comment]
rss订阅插件只能显示 tip 中的内容，正文不能显示，
作者你的 vuepress-theme-reco 有些可以，有些不可以，大佬能看下不
![image](https://user-images.githubusercontent.com/43113063/102864800-ba255700-446f-11eb-8865-612fc41ceee3.png)
![image](https://user-images.githubusercontent.com/43113063/102864809-be517480-446f-11eb-86a6-555faa87377e.png)
